### PR TITLE
Added my profile info

### PR DESCRIPTION
**Pull request by yashwankhade**

### CODE REVIEW

File: public/data/yashwankhade.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
+    "name": "Yash Wankhade",
+    "bio": "Hello, I am a web developer and an open source enthusiast. ",
```
Comment (Panquesito7) [inline public/data/yashwankhade.json:3]:
```suggestion
    "bio": "Hello, I am a web developer and an open source enthusiast",
```

File: public/data/yashwankhade.json [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    "avatar": "https://github.com/yashwankhade.png",
+    "links": [
+      {
+        "name": "Follow me on GitHub",
+        "url": "http://github.com/yashwankhade",
```
Comment (Panquesito7) [inline public/data/yashwankhade.json:8]:
```suggestion
        "url": "https://github.com/yashwankhade",
```

File: public/data/yashwankhade.json [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+        "icon": "github"
+      },
+      {
+        "name": "Follow me on Twitter",
+        "url": "http://twitter.com/CodingisCoool",
```
Comment (Panquesito7) [inline public/data/yashwankhade.json:13]:
```suggestion
        "url": "https://twitter.com/CodingisCoool",
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]
One question: did you modify the `package-lock.json` file manually?

**Comment by yashwankhade** [comment]
Nope I didn't change anything in ```package-lock.json```. When I had forked, that time itself showed me some changes in the code.

**Comment by Panquesito7** [comment]
> Nope I didn't change anything in `package-lock.json`. When I had forked, that time itself showed me some changes in the code.

Next time, do not commit `package-lock.json`, as we only want to add your profile in this PR. 🙂

**Comment by yashwankhade** [comment]
> > Nope I didn't change anything in `package-lock.json`. When I had forked, that time itself showed me some changes in the code.
> 
> Next time, do not commit `package-lock.json`, as we only want to add your profile in this PR. 🙂

Yeah sorry, I forgot to commit it via GitHub desktop, there I could have unchecked that file, so what can be done now?

**Review by Panquesito7 — CHANGES_REQUESTED** [review]
Could you please revert the changes in the `package-lock.json` file? Thank you.

**Pull request closed without merging**